shopfloor/tests/test_menu_counters.py

```diff
@@ -9,8 +9,8 @@ def test_menu_search(self):
             self.menu1.id: {
                 "lines_count": 2,
                 "picking_count": 2,
-                "priority_lines_count": 2,
-                "priority_picking_count": 2,
+                "priority_lines_count": 0,
+                "priority_picking_count": 0,
```

Comment:
I suppose we should have 2 priority lines and 1 priority picking related to the `menu1` here based on the configuration of pickings in `test_menu_base.py` above, link to my previous comment https://github.com/OCA/wms/pull/247/files#r675964910

Reply:
So the result is OK here: `picking1` and `picking3` are assigned but has a low priority (set to `"0"`), while the picking2 which is as urgent is still `confirmed` so it has no line counted here.